addons/l10n_fr_hr_holidays/models/hr_leave.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+from odoo import fields, models
+
+class HrLeave(models.Model):
+    _inherit = 'hr.leave'
+
+    l10n_fr_date_to = fields.Datetime('End Date For French Rules', readonly=True)
+
+    def _compute_fr_number_of_days(self, employee, date_from, date_to, employee_calendar, company_calendar):
```

Comment:
I know it's my fault but it is better to name it something else than `_compute..` to avoid confusion